Makefile

```diff
@@ -32,7 +32,7 @@ ISO := $(or $(ISO), "") # ISO should point to a file that has the '.iso' extensi
 NUM_MASTERS :=  $(or $(NUM_MASTERS),3)
 WORKER_MEMORY ?= 8892
 MASTER_MEMORY ?= 16984
-NUM_WORKERS := $(or $(NUM_WORKERS),0)
+NUM_WORKERS := $(or $(NUM_WORKERS),2)
```

Comment:
I wonder if we want to always test with two workers
it will add coverage, but will be harder to do on the local environment

Reply:
It's the default value moved from conftest.py.
@RazRegev wanted to make sure that there is a minimum of two workers for the e2e test but that doesn't work with SNO.

Reply:
it is bad idea. We don't want by default to run with 2 workers. It is on purpose set to 0 here

Reply:
It was minimum 2 in the code.
Maybe @slaviered or @RazRegev will can explain why it set to be minimum 2 (before SNO)

Reply:
Let's keep it a default of 0 for local deployment and 2 in the [CI](https://github.com/openshift/release/blob/5070635c0545d5bf786d5b35dca20b511a9e6eb1/ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh#L92)

Reply:
I think there is a misunderstanding, I don't recall wanting 2 workers, don't see any reason to do that